ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/commons/objectivefunctionevaluator/ObjectiveFunction.java

```diff
@@ -45,22 +46,22 @@ public Set<FlowCnec> getFlowCnecs() {
     }
 
     public Pair<Double, List<FlowCnec>> getFunctionalCostAndLimitingElements(FlowResult flowResult) {
```

Comment:
why can this one not be called with a flow result and a remedial action activation result?

Reply:
this is for margin optimization where only flow results are needed for evaluation, I figured it was easier to have a version of the method only using a FlowResult as it is used in a lot of parts of the code